GameData/RP-1/Contracts/Commercial Applications 2/FirstGeosync.cfg

```diff
@@ -0,0 +1,116 @@
+CONTRACT_TYPE
+{
+	name = FirstGeosync
+	title = Geosynchronous Test Satellite
+	group = CommApp2
+	agent = Federation Aeronautique Internationale
+
+	description = <b>Program: Advanced Commercial Applications<br>Type: <color=blue>Optional</color></b><br><br>With an orbital period of an earth day, a satellite will remain above a constant longitude, appearing to drift north and south through the day. This will allow for constant connection to a ground station.<br>Historical example: Syncom 2 (39kg, Thor-Delta) launched in July of 1963.<b>
+
+	synopsis = Launch a geosynchronous test satellite
+
+	completedMessage = Congratulations! The satellite is in orbit and connected to the groundstation.
+
+	sortKey = 409
+
+	cancellable = true
+	declinable = false
+	autoAccept = false
+	minExpiry = 0
+	maxExpiry = 0
+	maxCompletions = 1
+	maxSimultaneous = 1
+	deadline = 0
+
+	targetBody = HomeWorld()
+
+
+	// ************ REWARDS ************
+	prestige = Trivial       // 1.0x
+	advanceFunds = 0
+	rewardScience = 0
+	rewardFunds = 0
+	failureFunds = 0
+	rewardReputation = 40
```

Comment:
Not sure it's fair to give this contract a lower reward than the optional Molniya and Tundra contracts.